docs/DeveloperGuide.adoc

```diff
@@ -190,7 +192,32 @@ DD/MM/YYYY.
 
 * `AddEarningsCommandParser` would then return a `AddEarningsCommand` object with the parameter, `Earnings` object.
 
-*Example*
+The following activity diagram summarizes what happens when a user executes a new command.
+
+image::AddEarningsActivityDiagram.png[]
```

Comment:
The branching node (input is valid) only has one branch. Maybe include a second arrow for else that goes to the end of the diagram? Also, "input is valid" should be contained within square brackets ([input is valid]).

Reply:
Design diagrams have been unchanged since AB3. From how it looked like in Implementation, I would assume that there have been some changes to the architecture? I could be mistaken, of course.